packages/botonic-react/src/utils.js

```diff
@@ -8,6 +11,47 @@ import { Video } from './components/video'
 import { Document } from './components/document'
 import { Location } from './components/location'
 
+export function decomposeComponent(component) {
```

Comment:
"component" argument is actually a children array, right?
I think it's not obvious what/why this method does.
Would childrenToJson be a good name?
Would it make sense to create a ComponentJSON class with a to toJson and fromJson (with the msgToBotonic code) methods?

Reply:
I think childrenToJson it's not a good name, because in this case we are giving a children Component to decompose it into JSON format, but not always has to be a children. For example, in the future, we maybe want to transform a component into a componentJSON, not always has to be a children.

I think It's a good approach to create a ComponentJSON in order to encapsulate all this logic, I'll add it for a new PR!

Reply:
ok, thanks